### PR TITLE
[fr] create fr-Onboarding.md

### PR DESCRIPTION
**Pull request by gegeweb**
Translation of `en-Onboarding.md`with screenshot from the French GUI.

Ready for review.

/ping @trebmuh @ann0see

### CODE REVIEW

File: wiki/fr/fr-Onboarding.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Pour que les autres sachent qui vous êtes, vous devez configurer votre *profil*. Pour le faire, cliquez sur « Vue » (barre des menus) > « Mon profil… ».  
+Maintenant vous allez voir :
+
+![Profil de musicien](https://user-images.githubusercontent.com/9108457/101895791-48fcbe80-3ba8-11eb-913e-d0e4e7d60666.png)
+
+Remplissez au minimum votre « Pseudo/nom » pour renseigner votre nom et fermez la fenêtre. 
+
+### 2. Connectez vous à un serveur
+Maintenant que vous êtes configuré, vous pouvez utiliser le bouton « Se connecter » dans la fenêtre principale de Jamulus pour rejoindre un serveur. Une fenêtre s'ouvre alors dans laquelle vous voyez les *serveurs regroupés par genre*. Pour choisir un autre genre, sélectionnez-en un dans la liste en haut à gauche. Après avoir sélectionné un genre et un serveur, rejoignez le serveur en cliquant sur « Se connecter ». Si vous rencontrez des problèmes ici, ils sont très probablement dus à un équipement audio mal configuré ou à des problèmes de réseau. Consultez la [page de dépannage du client](Client-Troubleshooting).
+
```
Comment (ann0see) [inline wiki/fr/fr-Onboarding.md:33]: Do you need a full stop here: `de réseau. Consultez` in french? Could we maybe make it one sentence (If that isn't the case in the english translation, we should fix it there too.)

File: wiki/fr/fr-Onboarding.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Aide et communauté
+Si vous avez besoin d'aide ou voulez contribuer au pojet Jamulus, vous pouvez consulter les liens suivants :
+
+### Forums de discussion
+Pour de l'aide, et des discussions générales à propos de Jamulus : jetez un œil aux [forums sur Sourceforge](https://sourceforge.net/p/llcon/discussion/) (en anglais).
+
+### Développement et contribution
+Si vous voulez contribuer au logiciel Jamulus, jetez un œil au [dépot Github de Jamulus](https://github.com/corrados/jamulus/) et lisez le fichier contribution qui s'y trouve.
+
```
Comment (ann0see) [inline wiki/fr/fr-Onboarding.md]: I think we could put the Facebook groups here. Maybe we should open a PR to the english version. But this is to be discussed.

**Review by ann0see — COMMENTED** [review]
A few comments. But not a real review.

File: wiki/fr/fr-Onboarding.md (hunk 1 above)
Comment (gegeweb) [reply wiki/fr/fr-Onboarding.md:33]: In the EN version:
_[…]wrongly configured audio gear or network issues. Have a look at the [Troubleshooting page](Client-Troubleshooting)._

We can use the both, it is understandable. I don't have a fixed opinion on this point.

File: wiki/fr/fr-Onboarding.md (hunk 2 above)
Comment (gegeweb) [reply wiki/fr/fr-Onboarding.md]: Why not, and perhaps we can link to the French Bastorius website here too (for the French version).
The links are no longer in the other pages (Getting started and installtion).

In the Help and Community paragraph I think.

File: wiki/fr/fr-Onboarding.md (hunk 1 above)
Comment (ann0see) [reply wiki/fr/fr-Onboarding.md:33]: Yeah. Ok. For me it sounded a bit strange but not sure what @gilgongo thinks about this (especially for the english version)

File: wiki/fr/fr-Onboarding.md (hunk 2 above)
Comment (ann0see) [reply wiki/fr/fr-Onboarding.md]: Yes. I know there was a discussion somewhere about adding facebook groups etc. but I don't know where it was. Do you know the issue?

**Comment by ann0see** [comment]
I assume this is the last PR for french and r3.6.2?

**Comment by gegeweb** [comment]
> I assume this is the last PR for french and r3.6.2?

Yes.

File: wiki/fr/fr-Onboarding.md (hunk 2 above)
Comment (gegeweb) [reply wiki/fr/fr-Onboarding.md]: > Do you know the issue?

No… there is a lot of discussions! ;)

**Comment by ann0see** [comment]
Thanks for the information. I suggest, that after this is finished, we do a few spell checks in our languages. After that we can probably release it.

**Comment by trebmuh** [comment]
Review is done and this PR is merged.
That was a good translation.

**Pull request merged**